recipes/tsne/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tsne" %}
+{% set version = "v0.1.9" %}
+{% set sha256 = "1d41ebfc24494f949d87bd137eb68760179db80f6b70ac7be309dc93f9a2b296" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/WiscEvan/tsne/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  skip: false  # [py==27]
```

Comment:
Please remove this line, it is not necessary
```suggestion
```

Reply:
I thought `# [py<37]` would include 3.7 as well as 3.8. Is this incorrect?

@danielfrg, an updated release would be useful. I have removed `extra_link_args = ['-lsatlas']`  from the linux distribution checks within `setup.py`, the tests seem to pass with warnings using pytest. I can submit these changes to you as a PR from my repo if necessary.